docs/user-guides/deployment.md

```diff
@@ -0,0 +1,245 @@
+# Deployment
+
+There are two major ways for the deployment of the Vald cluster ways: Using the Helm command with `values.yaml` or `vald-helm-operator`.
+
+- Using Helm command
+    - Easy to deploy
+    - Need to Helm command when updating configuration
+
+- Using `vald-helm-operator`
+    - Monitoring the Vald deployments and Vald custom resource by `vald-helm-operator`
+    - Use `kubectl` command to set or update of the Vald cluster configuration
+    - Automate manage the Vald cluster based on CRD (Custom Resource Definitions)
+
+## Requirement
+
+- Helm: v3 ~
+
+If Helm or HDF5 is not installed, please install [Helm](https://helm.sh/docs/intro/install).
+
+<details><summary>Installation command for Helm</summary><br>
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+```
+
+</details>
+
+## Before Deployment
+
+Both ways need the configuration file when you need to overwrite the default configurations.
```

Comment:
```suggestion
Both ways need to configure the configuration file before the deployment when you need to overwrite the default configurations.
```

Reply:
```suggestion
Both ways need the configuration file when you need to overwrite the default configurations before the deployment.
```
It'd would be better.
`to configure the configuration file` is a little redundant.